spec/mailers/user_mailer_spec.rb

```diff
@@ -29,7 +29,7 @@
     end
 
     it 'assigns confirmation_url for email body' do
-      @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
+      @mail.body.encoded.should match("https://localhost:3000/g/#{@group.key}")
```

Comment:
I don't quite follow. I followed a red-green-refactor format. I wrote a
test which I that feels like it's of the form :

Given env is set to FORCE_SSL
When an email with link is generated
Then it has https links

I haven't written a test for the negative case, but i did see it behaving.
I don't know how to stub ENV, I did research it.

Can we talk about where to next on this



On Mon, 9 Mar 2015 at 17:49 James Kiesel <notifications@github.com> wrote:

> In spec/mailers/user_mailer_spec.rb
> <https://github.com/loomio/loomio/pull/1974#discussion_r26015281>:
>
> > @@ -29,7 +29,7 @@
> >      end
> >
> >      it 'assigns confirmation_url for email body' do
> > -      @mail.body.encoded.should match("http://localhost:3000/g/#{@group.key}")
> > +      @mail.body.encoded.should match("https://localhost:3000/g/#{@group.key}")
>
> I'm not sure that this test is actually testing your change, since you're
> just setting a default in the test environment and then seeing that it
> comes through.
>
> ENV['FORCE_SSL'] = true
> expect(@mail.body.encoded).to match /https/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1974/files#r26015281>.
>